### DISCUSSION TITLE
How to pass lexical the HTML converters from custom features

### DISCUSSION THREAD

**Discussion by thenoahhein** [payloadcms/payload#4681] [Q&A]
Hey everyone! I'm developing a few additions to the lexical editor, and it seems like it's working. My feature shows up in the slash menu, and works in the editor. But having followed some of the payload examples, my `richText` component looks something like this

```jsx
import React from 'react'

import { serializeLexical } from './serialize'

const RichText: React.FC<{ className?: string; content: any }> = ({ content }) => {
  if (!content) {
    return null
  }

  return (
    <div className='prose dark:prose-invert xl:prose-lg sm:prose-sm'>
      {content &&
        !Array.isArray(content) &&
        typeof content === 'object' &&
        'root' in content &&
        serializeLexical({ nodes: content?.root?.children })}
    </div>
  )
}

export default RichText
```

and the serializer looks like the ones from the payload repos. It seems this approach doesn't pick up any of the html converters from the features, and instead relies on the serializer to convert all of the lexical nodes to html. Looking through the docs I didn't really find what I was looking for. It looks like some of the bits and pieces are there, but I don't really know how I should be putting it all together.

**Comment by thompsonsj** [comment]
Can you paste the content of `serialize`?
